concordia/models.py

```diff
@@ -84,16 +84,41 @@ def get_absolute_url(self):
         return reverse("transcriptions:campaign-detail", args=(self.slug,))
 
 
+class Topic(models.Model):
+    objects = PublicationQuerySet.as_manager()
+
+    published = models.BooleanField(default=False, blank=True)
+
+    ordering = models.IntegerField(
+        default=0, help_text="Sort order override: lower values will be listed first"
+    )
+    title = models.CharField(blank=False, max_length=255)
+    slug = models.SlugField(blank=False, allow_unicode=True, max_length=80)
+    description = models.TextField(blank=True)
+    thumbnail_image = models.ImageField(
+        upload_to="topic-thumbnails", blank=True, null=True
```

Comment:
We should probably ticket using a separate storage bucket for our non-collection media at some point since we're accumulating more of them over time.

Reply:
So then we can't use crowd-media to point to it?